### PR TITLE
Upgrade node-forge to 0.10.0

### PR DESCRIPTION
**Pull request by davimacedo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "debug": "^3.1.0",
     "jsonwebtoken": "^8.1.0",
-    "node-forge": "^0.7.1",
+    "node-forge": "^0.10.0",
```
Comment (dplewis) [inline package.json:23]: Can you pin the packages?

**Review by dplewis — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (davimacedo) [reply package.json]: done

**Comment by davimacedo** [comment]
I updated all dependencies to the latest version but the tests are failing for node 8. I believe we should support only node >= 10. Thoughts? @dplewis

**Comment by Ashoat** [comment]
@davimacedo, any context as to why you closed this? There appears to be a [security issue](https://github.com/advisories/GHSA-92xj-mqp7-vmcj) with the old `node-forge`, so would be great to update it.

**Comment by TomWFox** [comment]
@Ashoat Davi opened a new PR #19

**Comment by Ashoat** [comment]
Ah yeah #20, I missed that. Thanks!

**Pull request closed without merging**